src/index.css

```diff
@@ -1,27 +1,50 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  transition: 0.8s;
+  margin: 0 0;
+}
+
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  background-color: #dedbd8;
+}
+
+#root {
```

Comment:
- [ ] Let's try to make the calculator as similar as possible to the wireframe provided. The most noticeable difference is the padding on the right of the buttons, which should be the same as for the screen.

This is yours:
![image](https://user-images.githubusercontent.com/79114233/144118996-ae66db33-a537-4681-ae1e-c0b1b64adaff.png)

This is the wireframe:
![image](https://user-images.githubusercontent.com/79114233/144119033-d421c714-83a3-4b3c-be40-4619a3d20248.png)

Reply:
I'm sure what screen you're using but I cant seem to see what the issue is on your side.. I use grid, and with is equal for both the screen so I'm sure why your screen is acting like that.
<img width="381" alt="Screen Shot 2021-11-30 at 22 12 02" src="https://user-images.githubusercontent.com/30793016/144121356-01f784e0-06b1-4614-b183-0d4240fdda62.png">

Reply:
Please check again.. or use another browser